src/ToggleButton/index.jsx

```diff
@@ -0,0 +1,182 @@
+import React               from 'react';
+import PropTypes           from 'prop-types';
+
+import { buildClassName }  from '../utils';
+import styles              from './toggleButton.css';
+import Icon                from '../Icon/';
+
+const ToggleButton = ( {
+    children,
+    className,
+    cssMap,
+    iconPosition,
+    iconSize,
+    iconType,
+    id,
+    isDisabled,
+    isChecked,
+    isReadOnly,
+    label,
+    onBlur,
+    onClick,
+    onFocus,
+    onMouseOut,
+    onMouseOver,
+    role
+
+} ) =>
+{
+    const clickProp = !isReadOnly ? onClick : false;
```

Comment:
This is unnecessary because the `click` event will not fire on a `<button>` that has the `disabled` attribute anyways.

You can remove this line. Below we can simply do:
```
...
onClick = { onClick }
...
```